api/redhatopenshift/HcpCluster.Management/hcpCluster-models.tsp

```diff
@@ -196,8 +196,17 @@ model ApiProfile {
   /** The internet visibility of the OpenShift API server */
   @visibility(Lifecycle.Read, Lifecycle.Create)
   visibility?: Visibility = Visibility.Public;
+
+  /** The list of authorized IP ranges (IPv4 CIDRs) allowed to access the API server. Maximum 5000 entries. */
+  @visibility(Lifecycle.Read, Lifecycle.Create, Lifecycle.Update)
+  @maxItems(5000)
+  authorizedIpRanges?: ipCidr[];
```

Comment:
There was a suggestion in https://github.com/Azure/ARO-HCP/pull/1932 to rename this to `authorizedCIDRs`.

Reply:
We are using authorizedCidrs now. Thanks!